web-client/src/app/views/pay/pay.page.stories.ts

```diff
@@ -29,8 +31,3 @@ const Template: Story<Args> = ({ receiverAddress, ...state }: Args) => ({
 });
 
 export const Default = Template.bind({});
-Default.args = {
-  name: 'Test Owner',
-  balance: 1234,
-  receiverAddress: 'G6AIRDAJFSBXNFBHLQ2F5JLZJ6EYYYLDZSCDHUQUB2YUG5QO4ZB4VNAL7I',
-};
```

Comment:
Can we re-use the extract component story here, so that the story still renders with the placeholders like before? (See the [Chromatic change / comment](https://www.chromatic.com/test?appId=619e482ab3e38b003a063523&id=619e4abeae73be003a7189ba))

The Storybook docs cover some of the ways to approach this: https://storybook.js.org/docs/angular/writing-stories/stories-for-multiple-components